app/views/passwords/forgot_password.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.forgot_password.container
+  .row
+    .col-sm-10.col-bg-offset-1
+      = form_tag request_token_path, class: "sign_in" do
```

Comment:
Be careful sharing classes between pages. You could add some styling to the sign in page and inadvertently have it apply to this page as well. Sharing classes is fine as long as you keep in mind where you have used the class.